mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterActor.java

```diff
@@ -495,6 +495,7 @@ private void onTaskExecutorRegistration(TaskExecutorRegistration registration) {
             }
             sender().tell(Ack.getInstance(), self());
         } catch (Exception e) {
+            log.error("onTaskExecutorRegistration exception caught", e);
```

Comment:
we don't need this log.error since this is already being emitted when connection fails.